app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -82,13 +83,49 @@ import org.mozilla.fenix.theme.FirefoxTheme
  */
 @Composable
 @Suppress("LongParameterList")
-fun RecentTabs(
+fun JumpBackIn(
```

Comment:
Hm.. my original intention of having the 2 separate components was to dissolve having a @Composable that needed to take the combination of the `RecentSyncedTabState` and `List<RecentTab>` states and their respective callbacks since we can just pass these parameters to the components directly in `Content()` instead of having a wrapper component.

Reply:
Ah, I see. I'll update that. I think it makes more sense to remove the `SyncedTab` from the `RecentTab` sealed class as well, so I'll do some further refactoring.